### PR TITLE
Enable some compiler flags on Swift 5.9+

### PR DESCRIPTION
**Pull request by MahdiBM**
Enable the `ExistentialAny` and `StrictConcurrency` compiler flags.

- [x] Tests passed locally.

### CODE REVIEW

File: Package@swift-5.9.swift [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// swift-tools-version: 5.9
+
+import PackageDescription
+
+let swiftSettings: [SwiftSetting] = [
+    .enableUpcomingFeature("ExistentialAny"),
```
Comment (0xTim) [inline Package@swift-5.9.swift]: Won't we need this in 5.8 to make the change compile?

Comment (MahdiBM) [reply Package@swift-5.9.swift]: No, it works down until 5.7.
But I think we might be able to just drop 5.8 all together, the generator is already at 5.9+.
I need to check.

Comment (0xTim) [reply Package@swift-5.9.swift]: Yeah if the generator is 5.9 we can move to 5.9 only

Comment (MahdiBM) [reply Package@swift-5.9.swift]: Sorry that was some misinformation, the generator is still on 5.8.
But again, it won't cause problems on 5.8.

Comment (MahdiBM) [reply Package@swift-5.9.swift]: Alright i've updated the PR to be 5.9-only in anticipation of the 1.0.0 swift-openapi-generator release.

**Review by 0xTim — APPROVED** [review]
Thanks!

**Pull request merged**